packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -536,7 +544,7 @@ const requestProcessor =
                     libraryTarget: 'commonjs2'
                 },
                 // use eval-source-map for server-side debugging
-                devtool: mode === development ? 'eval-source-map' : false,
+                devtool: mode === development && INSPECT ? 'eval-source-map' : false,
```

Comment:
Wait, would this prevent source-map for client? when we are not starting the app with inspect mode?

Reply:
The request processor is something that runs on the "server"... well at least on the browser. So if you wanted to debug it, you'd be adding the `--inspect` flag anyway.